big-o-notation/big-o.js

```diff
@@ -0,0 +1,47 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {}; // 1 * 1 = O(1)
+  const unique = []; // 1 * 1 = O(1)
+  for (
+    let i = 0; // 1 * 1 = O(1)
+    i < words.length; // 2 * n = O(n)
+    i++ // 2 * n = O(n)
+  ) {
+    const word = words[i]; // 2 * n = O(n)
+    if (!seen[word]) {
+      // 2 * n = O(n^2)
+      seen[word] = true; // 2 * n = O(n^2)
+      unique[unique.length] = word; // 3 * n = O(n^3)
+    }
+  }
+  return unique; // 1 * 1 = O(1)
+} // Big O Notation for uniqueLinear: O(n^3)
```

Comment:
You correctly pulled out the largest value, but since those are wrong, this should actually just be O(n) (linear).